.evergreen/releases.yml

```diff
@@ -40,45 +74,82 @@ functions:
           rm expansion.yml
 
   "install dependencies":
-    command: shell.exec
-    params:
-      working_dir: "src"
-      script: |
-        ${PREPARE_SHELL}
-        .evergreen/install-dependencies.sh rust
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          .evergreen/install-dependencies.sh rust
+
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        include_expansions_in_env:
+          - DRIVERS_TOOLS
+        binary: bash
+        args:
+          - .evergreen/fetch-drivers-tools.sh
+
+  "build papertrail vars":
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        include_expansions_in_env:
+          - DRIVERS_TOOLS
+          - DRY_RUN
+          - GIT_TAG
+        binary: bash
+        args:
+          - .evergreen/release-build-papertrail-vars.sh
+
+    - command: expansions.update
+      params:
+        file: src/papertrail-expansion.yml
+
+    - command: shell.exec
+      params:
+        working_dir: "src"
+        script: rm papertrail-expansion.yml
 
   "fetch tag":
     command: subprocess.exec
     params:
       working_dir: "src"
-      add_expansions_to_env: true
+      include_expansions_in_env:
+        - GIT_TAG
       binary: bash
       args:
         - .evergreen/release-fetch-tag.sh
 
   "publish release":
-    - command: shell.exec
+    - command: subprocess.exec
       type: test
       params:
         working_dir: "src"
-        script: |
-          set +x
+        add_expansions_to_env: true
```

Comment:
Unfortunately, the release script uses `.evergreen/env.sh`, which uses a whole giant pile of vars.